HTML/1. intro.md

```diff
@@ -0,0 +1,39 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language that is used to create webpages, so it's not like a programming language like Python
+___
+HTML is structured by tags and content which make up elements. There ways of declaring an element.
+
+1. The first way of declaring elements is with an opening tag and a closing tag, the opening tag has a `<` to begin and a `>` to end. The closing tag has a `</` to begin and a `>` to end. Everything else between the opening and closing tags is either content or other nested elements
+
+
+2. The second way is through self closing tags, that don't have any content between them and it is structured with a `<` to begin and a `>` to end. They're also called void elements and what that means is that it can't contain any nested Elements, Comments, and any Character data.
```

Comment:
Can also end with /> I am pretty sure... or it starts with </

Reply:
An element can end with `/` just as much as `/` can be an attribute name: it's only allowed because HTML normalizes all errors.

`<a /="b">` - clearly incorrect.

The `/` in `/>` is in a boolean attribute position.